client/my-sites/media-library/test/list-item-image.jsx

```diff
@@ -31,26 +27,26 @@ describe( 'MediaLibraryListItem image', () => {
 
 	describe( 'thumbnail display mode', () => {
 		test( 'defaults to photon when no thumbnail parameter is passed', () => {
-			wrapper = shallow( getItem( 0 ) );
+			render( getItem( 0 ) );
 
-			expect( wrapper.props().src ).toEqual( getResizedUrl() );
+			expect( screen.getByTestId( getResizedUrl() ) ).toBeInTheDocument();
```

Comment:
Should we use `toBeVisible` as a more specific version of `toBeInTheDocument()`?